bootstrap/Dockerfile

```diff
@@ -37,7 +39,14 @@ RUN cd /opt/ksonnet && \
 
 RUN mkdir -p /opt/kubeflow
 
-COPY bootstraper /opt/kubeflow/
+ENV PATH /go/bin:/usr/local/go/bin:$PATH
+
+RUN mkdir -p $GOPATH/src/github.com/kubeflow && \
+    cd $GOPATH/src/github.com/kubeflow && \
+    git clone $KUBEFLOW_REPO
+
+RUN go build -i -o /opt/kubeflow/bootstraper ${GOPATH}/src/github.com/kubeflow/kubeflow/bootstrap/cmd/bootstrap/main.go
```

Comment:
How about using multi-stage builds so the final image doesn't include the source tree?
https://docs.docker.com/develop/develop-images/multistage-build/#use-multi-stage-builds

Reply:
Good point, updated